load.py

```diff
@@ -1,4 +1,13 @@
+import sys
```

Comment:
So, every time the code in this module is executed, we remove all the package modules from `sys.modules` except for the `load.py` module

Instead of exposing this as a build system, can we just expose it via the command palette by putting it in `Default.sublime-commands`?

Reply:
Yes and yes.

I not even thought about this as I *always* use it as a build system but it is a normal ST command after all.

Reply:
Of course if you add/mangle `sys.path` beyond that manually these operations need to be indempotent.  I think that's already the case though.